cmd/util/ledger/migrations/storage_v6_test.go

```diff
@@ -787,3 +789,166 @@ func uint64ToBinary(integer uint64) []byte {
+		"storage\x1fdapperUtilityCoinReceiver":                      "00cade0005d88484d8c08248e544175ee0461c4b6f546f6b656e466f7277617264696e6702846475756964d8a41a0348741769726563697069656e74d8c983d88348ead892083b3e2c6cd8c8820378196461707065725574696c697479436f696e5265636569766572f67819546f6b656e466f7277617264696e672e466f72776172646572",
+		"public_key_5":                                              "f847b840dac22f8bdf651e21f10d5cc96ef8ca9a55bd28b56b117dda0877b3bce76ce7e04c4fe330c48e5c81f13f76c7dc1b86ab49a33d8b2498cbbed24459f7e1e918c50201808080",
+		"public\x1fprivateForwardingPublic":                         "00cade0005d8cb82d8c88201781870726976617465466f7277617264696e6753746f72616765d8db82f4d8d582d8c0824818eb4ee6b3c026d27818507269766174655265636569766572466f727761726465727822507269766174655265636569766572466f727761726465722e466f72776172646572",
+		"exists":                                                    "01",
+		"public\x1fdapperUtilityCoinReceiver":                       "00cade0005d8cb82d8c8820178196461707065725574696c697479436f696e5265636569766572d8db82f4d8dc82d8d582d8c08248ead892083b3e2c6c714461707065725574696c697479436f696e774461707065725574696c697479436f696e2e5661756c7481d8d682d8c08248f233dcee88fe0abe6d46756e6769626c65546f6b656e7646756e6769626c65546f6b656e2e5265636569766572",
+		"storage\x1fMomentCollection\x1fownedNFTs\x1fv\x1f8186472":  "00cade0005d88484d8c082480b2a3299cc857e2967546f7053686f7402866464617461d88484d8c082480b2a3299cc857e2967546f7053686f74018666706c61794944d8a31903b16c73657269616c4e756d626572d8a3190f9a657365744944d8a3181a72546f7053686f742e4d6f6d656e7444617461626964d8a41a007cea686475756964d8a41a019776ec6b546f7053686f742e4e4654",
+		"private\x1fflowTokenReceiver":                              "00cade0005d8cb82d8c882016e666c6f77546f6b656e5661756c74d8db82f4d8dc82d8d40581d8d682d8c08248f233dcee88fe0abe6d46756e6769626c65546f6b656e7646756e6769626c65546f6b656e2e5265636569766572",
+		"storage\x1fMomentCollection\x1fownedNFTs\x1fv\x1f9061281":  "00cade0005d88484d8c082480b2a3299cc857e2967546f7053686f7402866464617461d88484d8c082480b2a3299cc857e2967546f7053686f74018666706c61794944d8a31903e96c73657269616c4e756d626572d8a3194a6a657365744944d8a3181a72546f7053686f742e4d6f6d656e7444617461626964d8a41a008a43a16475756964d8a41a01b2beca6b546f7053686f742e4e4654",
+		"public_key_3":                                              "f847b840dac22f8bdf651e21f10d5cc96ef8ca9a55bd28b56b117dda0877b3bce76ce7e04c4fe330c48e5c81f13f76c7dc1b86ab49a33d8b2498cbbed24459f7e1e918c50201808080",
+		"public_key_7":                                              "f847b840dac22f8bdf651e21f10d5cc96ef8ca9a55bd28b56b117dda0877b3bce76ce7e04c4fe330c48e5c81f13f76c7dc1b86ab49a33d8b2498cbbed24459f7e1e918c50201808080",
+		"public_key_1":                                              "f847b840dac22f8bdf651e21f10d5cc96ef8ca9a55bd28b56b117dda0877b3bce76ce7e04c4fe330c48e5c81f13f76c7dc1b86ab49a33d8b2498cbbed24459f7e1e918c50201808080",
```

Comment:
we might only keep one of these keys and update public_key_count to 1, they don't play that much role in this test

Reply:
Given that it doesn't hurt, I think keeping this real-world data might be better